Gruntfile.coffee

```diff
@@ -810,6 +825,7 @@ module.exports = (grunt) ->
         '!' + devBuildPath + '/integration/**/*.ts'
         '!' + devBuildPath + '/**/*.core-env.ts'
         '!' + devBuildPath + '/**/*.core-env.spec.ts'
+        '!' + devBuildPath + '/generic_core/monkey_proof/*.ts'
```

Comment:
I'd rather see this list shortening - and there's something weird about doing dependency injection at the source file level.

Swapping in and out freedomjs manifests has always seemed reasonably neat to me - since settings are just JSON, can you think of another way of getting a default config into each build?

(Also, some background: we'll soon eliminate this pre-copy and instead have the TypeScript compiler read directly from `src/` and output to `build/` - I don't think this will work well in that world.)